Doesn't build in Travis, does build on laptop

In file included from /home/amigo/ros/kinetic/system/src/problib/src/pdfs/Uniform.cpp:38:0:
/home/amigo/ros/kinetic/system/src/problib/include/problib/conversions.h:40:25: fatal error: problib/PDF.h: No such file or directory

It isn't able to find the generated msg header.

--- a/problib/CMakeLists.txt
+++ b/problib/CMakeLists.txt
@@ -63,8 +63,9 @@ add_library(problib
         include/${PROJECT_NAME}/pdfs/Hybrid.h
         include/${PROJECT_NAME}/pdfs/PMF.h)
 target_link_libraries(problib ${catkin_LIBRARIES})
+add_dependencies(problib ${${PROJECT_NAME}_EXPORTED_TARGETS})
+
 
 # add test executable
 add_executable(problib_test src/test.cpp)
 target_link_libraries(problib_test problib)
-add_dependencies(problib_test problib_generate_messages_cpp)

